src/solver/optimisation/opt_calcul_des_pmin_MUT_MDT.cpp

```diff
@@ -23,7 +23,6 @@
 
 #include "antares/solver/optimisation/opt_fonctions.h"
```

Comment:
### Don't treat this remark, it'll be done in a separate PR 

Same as above : 
We should replace : 

```c++
#include "antares/solver/optimisation/opt_fonctions.h"
#include "antares/solver/optimisation/opt_structure_probleme_a_resoudre.h"
#include "antares/solver/simulation/sim_extern_variables_globales.h"
#include "antares/solver/simulation/sim_structure_donnees.h"
#include "antares/solver/simulation/simulation.h"
``` 
with ; 
```c++
#include "antares/solver/simulation/sim_structure_probleme_economique.h"
```